mozetl/bhr_collection/bhr_collection.py

```diff
@@ -1242,7 +1242,13 @@ def etl_job_daily(sc, sql_context, config=None):
     help="Proportion of pings to use (1.0 is 100%)",
 )
 @click.option("--use_gcs", is_flag=True, default=False)
-def start_job(date, sample_size, use_gcs):
+@click.option(
+    "--thread-filter", default="Gecko", help="Specifies which thread we are processing hangs from"
+)
+@click.option(
+    "--output-tag", default="main", help="Tags the output filename"
+)
```

Comment:
Format changes for ci
```suggestion
@click.option(
    "--thread-filter",
    default="Gecko",
    help="Specifies which thread we are processing hangs from",
)
@click.option("--output-tag", default="main", help="Tags the output filename")
```